include/algorithms/weak_learner/weak_learner_model.h

```diff
@@ -51,7 +51,7 @@ namespace interface1
  * \snippet weak_learner/weak_learner_model.h Parameter source code
  */
 /* [Parameter source code] */
-class Parameter : public classifier::Parameter
+class Parameter : public classifier::interface1::Parameter
```

Comment:
Does it mean that Parameter class is deprecated now as there is no interface2::Parameter?

Reply:
weak_learner namespace is fully deprecated